docs/docsite/rst/playbook_guide/playbooks_loops.rst

```diff
@@ -274,6 +274,13 @@ You can combine the ``until`` keyword with ``loop`` or ``with_<lookup>``. The re
       delay: 1
       until: "uri_output.status == 200"
 
+Retrying a task with timeout
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When you use ``timeout`` in a loop, it applies per retry attempt. See :ref:`TASK_TIMEOUT <TASK_TIMEOUT>` for more details.
+
+If the task action execution exceeds the ``timeout``, the retry process is stopped.
```

Comment:
I'd say remove this because it is explained clearly in the `TASK_TIMEOUT` link in the preceding sentence.